java/ql/src/semmle/code/java/dataflow/FlowSources.qll

```diff
@@ -189,8 +187,7 @@ private class BeanValidationSource extends RemoteFlowSource {
     exists(Method m, Parameter v |
       this.asParameter() = v and
       m.getParameter(0) = v and
-      m
-          .getDeclaringType()
+      m.getDeclaringType()
```

Comment:
This formatting change is already in latest main, please rebase to avoid this change showing up here.

Reply:
This has been fixed while rebasing.